### PR TITLE
sdk-tutorials blog - fixing Handlers link reference

### PR DESCRIPTION
**Pull request by vinipx**
Quick adjustment for **Handlers** link reference to Cosmos SDK Docs

### CODE REVIEW

**Comment by Tosch110** [comment]
Hi @vinipx - thanks for submitting the PR. After checking the link you changed, it looks to me that the correct link sits here:
https://docs.cosmos.network/v0.39/building-modules/handler.html

(Since this tutorial is using version "launchpad" aka v0.39 - it should link to the same version in the docs)

File: blog/tutorial/01-index.md [hunk 1]
```diff
@@ -198,7 +198,7 @@ func (msg MsgCreatePost) ValidateBasic() error {
 
 Going back to `GetCmdCreatePost` in `x/blog/client/cli/tx.go`, you'll see `MsgCreatePost` being created and broadcast with `GenerateOrBroadcastMsgs`.
 
-After being broadcast, the messages are processed by an important part of the application, called [**handlers**](https://docs.cosmos.network/master/building-modules/handler.html).
+After being broadcast, the messages are processed by an important part of the application, called [**handlers**](https://docs.cosmos.network/master/building-modules/handler.html#handlers).
```
Comment (okwme) [inline blog/tutorial/01-index.md:201]:
```suggestion
After being broadcast, the messages are processed by an important part of the application, called [**handlers**](https://docs.cosmos.network/v0.39/building-modules/handler.html).
```

Comment (okwme) [reply blog/tutorial/01-index.md:201]: as per @Tosch110 's recommendation

Comment (vinipx) [reply blog/tutorial/01-index.md]: Fixed, thanks :)

**Review by Tosch110 — APPROVED** [review]

**Comment by vinipx** [comment]
@Tosch110 , could you please merge it? tks, buddy

**Pull request merged**